### ISSUE TITLE
Improve Instance-handling

### ISSUE THREAD

**Issue by PsypherPunk** [ukwa/w3act#278]
In the old ACT system Instances were handled slightly differently. In w3act an 'Instance' is any occurence of a seed in OpenWayback and created automatically.

Previously Instances were created by the job-handling script and less frequently—specifically, daily crawls will create 365 Instances a year for a Target which gets unwieldly very quickly; following discussion, we opted to only create one per week (on Mondays).

Something similar should probably be built into w3act. Needs discussion to work out the details as how best to handle this.

**Comment by anjackson** [comment]
I don't think this really requires a DB change. But we need to post-crawl script to notify ACT that there is a new instance, and this is supported by #260 - is that right @PsypherPunk?

**Comment by PsypherPunk** [comment]
Yes, I think so. The scripts we use are able to tell which Targets where in a given job so it should be a matter of giving us an API endpoint to poke.

I don't know whether #260 would require DB changes though, to add the additional metadata...?

**Comment by anjackson** [comment]
Yes, there are DB changes under that ticket:https://github.com/ukwa/w3act/issues/260#issuecomment-128310804
